go/kbfs/search/mapping.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package search
+
+import (
+	"github.com/blevesearch/bleve"
+	"github.com/blevesearch/bleve/analysis"
+	"github.com/blevesearch/bleve/analysis/char/html"
+	"github.com/blevesearch/bleve/analysis/token/lowercase"
+	"github.com/blevesearch/bleve/analysis/tokenizer/unicode"
+	"github.com/blevesearch/bleve/mapping"
+	"github.com/blevesearch/bleve/registry"
+)
+
+const (
+	htmlAnalyzerName = "html"
+	htmlFieldName    = "Html"
+)
+
+func htmlAnalyzerConstructor(
+	config map[string]interface{}, cache *registry.Cache) (
+	*analysis.Analyzer, error) {
+	tokenizer, err := cache.TokenizerNamed(unicode.Name)
+	if err != nil {
+		return nil, err
+	}
+	htmlFilter, err := cache.CharFilterNamed(html.Name)
+	if err != nil {
+		return nil, err
+	}
+	toLowerFilter, err := cache.TokenFilterNamed(lowercase.Name)
+	if err != nil {
+		return nil, err
+	}
+	rv := analysis.Analyzer{
+		Tokenizer: tokenizer,
```

Comment:
I wonder if we should add the `web` tokenizer too. Seems useful: https://github.com/blevesearch/bleve/blob/master/analysis/tokenizer/web/web.go

Reply:
Sure, it looks like it uses unicode under the covers and just adds a few exceptions.  I'll try it.